psql2mysql/__init__.py

```diff
@@ -281,6 +292,16 @@ def setupTypeDecorators(self, srcTable, targetTable):
                     targetTable.c[targetCol.name].type = decorator
 
 
+def positive_int(value):
```

Comment:
you can pass `min=0` to the `cfg.IntOpt('chunk-size', default=10000,                  help='Number of records to move per chunk.')` and it would make sure that the number is  positive without needing this extra func.

Reply:
oh cool, nice. I'll add a min. The only thing I don't like about that is it would be nice to have validation on the class not on when thinking about cli options. But it's apart of oslo.config and it required in the actual migration class so your probably right. shame, I liked the test.

Though if the migration class look in a chunk-size in the constructor, which it probably should rather then setting chunk-size in setup, we'd still need someway to validate the number.